Formula/nim.rb

```diff
@@ -14,28 +14,23 @@ class Nim < Formula
 
   head do
     url "https://github.com/nim-lang/Nim.git", branch: "devel"
-    resource "csources" do
-      url "https://github.com/nim-lang/csources.git"
-    end
   end
```

Comment:
We don't need a one-line `head` block. You can just put
```ruby
head "https://github.com/nim-lang/Nim.git", branch: "devel"
```
in the line after `license`.

Reply:
done